src/__main__.py

```diff
@@ -1,13 +1,23 @@
 #!/usr/bin/env python3
 """Main program"""
-from src.utils import config, config_ble, led_utils, get_logger
-from src.services import StatusIndicatorService
+import RPi.GPIO as GPIO
+from src.utils import config, config_ble, led_utils, get_logger, time_in_millisecond
```

Comment:
```suggestion
from src.utils import (
    config, 
    config_ble,
    led_utils, 
    get_logger, 
    time_in_millisecond
)
```

Histoire d'avoir partout les même types d'import.